score/README.md

```diff
@@ -1,49 +1,65 @@
-# Justice 40 Score generator
+# Justice 40 Score application
 
-## Setup
+## Running using Docker
 
-- Start a terminal
-- Make sure you have Python 3.9 installed: `python -V` or `python3 -V`
-- Create a `virtualenv` in this folder: `python -m venv venv`
-- Activate the virtualenv
-  - Windows: `./venv/Scripts/activate`
-  - Mac/Linux: `source venv/bin/activate`
-- Install packages: `pip install -r requirements.txt`
-  - If you are a Windows user, you might need to install Build Tools for Visual Studio. [Instructions here](https://stackoverflow.com/a/54136652)
+We use Docker to install the necessary libraries in a container that can be run in any operating system.
 
-## Running the Jupyter notebook
+To build the docker container the first time, make sure you're in the root directory of the repository and run `docker-compose build`
 
-- Start a terminal
-- Change to this directory (i.e. `cd score`)
-- Activate your virtualenv (see above)
-- Type `jupyter notebook`. Your browser should open with a Jupyter Notebook tab
+After the container is built, you can start the container wtih `docker-compose up`.
 
-## Activating variable-enabled Markdown for Jupyter notebooks
+To run commands, open a new terminal window or tab:
 
-- Change to this directory (i.e. `cd score`)
-- Run `jupyter contrib nbextension install --user`
-- Run `jupyter nbextension enable python-markdown/main`
-- Make sure you've loaded the Jupyter notebook in a "Trusted" state. (See button near
-  top right of Notebook screen.)
+- Get help: `docker exec j40_score_1 python3 application.py --help`
+- Clean up the data directories: `docker exec j40_score_1 python3 application.py data-cleanup`
 
-For more information, see [nbextensions docs](https://jupyter-contrib-nbextensions.readthedocs.io/en/latest/install.html) and
-see [python-markdown docs](https://github.com/ipython-contrib/jupyter_contrib_nbextensions/tree/master/src/jupyter_contrib_nbextensions/nbextensions/python-markdown).
+## Local development
+
+You can run the Python code locally to develop, using Poetry. However, to generate the census data you will need the [GDAL library](https://github.com/OSGeo/gdal) installed locally. Also to generate tiles for a local map, you will need [Mapbox tippeanoe](https://github.com/mapbox/tippecanoe)
+
+- Start a terminal
+- Make sure you have Python 3.9 installed: `python -V` or `python3 -V`
+- We use [Poetry](https://python-poetry.org/). Please follow the instructions to download to your system
+- Change to this directory (i.e. `cd score`)
+- Start a Poetry shell with `poetry shell`
```

Comment:
Add `poetry install` right before this?

Reply:
Also, do we need to run `poetry shell` in this list of steps? Is this necessary before going to the next section? Note that the next section uses `poetry` in the commands so seems like it assumes you're not in a poetry shell.

Reply:
You're absolutely right -- once in a shell you can run the commands without the `poetry` in front. You can also do `poetry run [cmd]` without the shell. I'll update for folks to use the latter.

Reply:
Updated.